Cargo.toml

```
@@ -1,8 +1,16 @@
 [package]
 name = "fastrand-contrib"
+# Make sure to modify CHANGELOG.md before releasing a new version.
 version = "0.1.0"
+authors = ["John Nunley <dev@notgull.net>", "Petr Nevyhoštěný <petr.nevyhosteny@gmail.com>"]
 edition = "2018"
 rust-version = "1.43.1"
```

```suggestion
rust-version = "1.43"
```

`rust-version = "1.43.1"` rejects build on 1.43.0, but there is no API that available on 1.43.1 but not on 1.43.0.